samples/msal-node-samples/auth-code-distributed-cache/src/AuthProvider.ts

```diff
@@ -102,80 +114,136 @@ export class AuthProvider {
+            .authority!.split("/")
+            .pop()!;
 
         try {
-            let [cloudDiscoveryMetadata, authorityMetadata] = await Promise.all([
-                cacheClient.get(`${clientId}.${tenantId}.discovery-metadata`),
-                cacheClient.get(`${clientId}.${tenantId}.authority-metadata`)
-            ]);
+            let [cloudDiscoveryMetadata, authorityMetadata] = await Promise.all(
+                [
+                    cacheClient.get(
+                        `${clientId}.${tenantId}.discovery-metadata`
+                    ),
+                    cacheClient.get(
+                        `${clientId}.${tenantId}.authority-metadata`
+                    ),
+                ]
+            );
 
             if (!cloudDiscoveryMetadata || !authorityMetadata) {
-                [cloudDiscoveryMetadata, authorityMetadata] = await Promise.all([
-                    AuthProvider.fetchCloudDiscoveryMetadata(tenantId),
-                    AuthProvider.fetchOIDCMetadata(tenantId)
-                ]);
+                [cloudDiscoveryMetadata, authorityMetadata] = await Promise.all(
+                    [
+                        AuthProvider.fetchCloudDiscoveryMetadata(tenantId),
+                        AuthProvider.fetchOIDCMetadata(tenantId),
+                    ]
+                );
 
                 if (cloudDiscoveryMetadata && authorityMetadata) {
-                    await cacheClient.set(`${clientId}.${tenantId}.discovery-metadata`, JSON.stringify(cloudDiscoveryMetadata));
-                    await cacheClient.set(`${clientId}.${tenantId}.authority-metadata`, JSON.stringify(authorityMetadata));
+                    await cacheClient.set(
+                        `${clientId}.${tenantId}.discovery-metadata`,
+                        JSON.stringify(cloudDiscoveryMetadata)
+                    );
+                    await cacheClient.set(
+                        `${clientId}.${tenantId}.authority-metadata`,
+                        JSON.stringify(authorityMetadata)
+                    );
                 }
             }
 
-            msalConfigWithMetadata.auth.cloudDiscoveryMetadata = typeof cloudDiscoveryMetadata === 'string' ? cloudDiscoveryMetadata : JSON.stringify(cloudDiscoveryMetadata);
-            msalConfigWithMetadata.auth.authorityMetadata = typeof authorityMetadata === 'string' ? authorityMetadata : JSON.stringify(authorityMetadata);
+            msalConfigWithMetadata.auth.cloudDiscoveryMetadata =
+                typeof cloudDiscoveryMetadata === "string"
+                    ? cloudDiscoveryMetadata
+                    : JSON.stringify(cloudDiscoveryMetadata);
+            msalConfigWithMetadata.auth.authorityMetadata =
+                typeof authorityMetadata === "string"
+                    ? authorityMetadata
+                    : JSON.stringify(authorityMetadata);
         } catch (error) {
-            console.log(error);
+            console.error(error);
```

Comment:
Don't catch. Or catch and rethrow.

Reply:
This is out of scope for this PR. We can address this in a follow-up PR.